tests/functions/test_alias.py

```diff
@@ -16,8 +18,8 @@ def test_alias_single():
     req = OriginRequest(conf_file=TEST_CONF)
 
     aliases = [
-        {"src": "/foo/bar", "dest": ""},
-        {"src": "/baz", "dest": "/quux"},
+        {"src": "/foo/bar", "dest": "", "exclude_paths": []},
+        {"src": "/baz", "dest": "/quux", "exclude_paths": []},
```

Comment:
Please be sure to test alias resolution for aliases without an `exclude_paths` key. None of the aliases in the current config (deployed to prod) contain an `exclude_paths` key.